Infrastructure/gpg-ipdeny/manifest.yml

```diff
@@ -2,8 +2,8 @@
 
 applications:
   - name: gpg-ipdeny
-    instances: 1
-    memory: 256M
+    instances: 3
```

Comment:
Is this on all environments? feels very high for dev, test and preprod - would've expected those to be 1 each. And we're going to autoscale live anyway right?

Reply:
Ah yeah, good point. We haven't deployed the IP deny app to test or preprod yet so I hadn't given them much thought - I'll take this back down to 1 for now!